service/templates/settings.html

```diff
@@ -89,10 +89,17 @@ <h3 style="padding-left:15px;"><i class="fa fa-globe"></i> User Settings</h3>
                   <input type="text" placeholder="octocat" ng-model="github_user" /> <label ng-show="{{ .GithubUID }} !== 0">Github UID Number: {{ .GithubUID }}</label>
               </div>
             </div>
+            <div class="form-group">
```

Comment:
Use the same css classes as the other fields above - it currently looks a bit off
<img width="856" alt="Screen Shot 2019-07-31 at 2 19 29 PM" src="https://user-images.githubusercontent.com/29384396/62237052-48da9500-b39e-11e9-81ec-2ad5d6e6b363.png">